intro_to_nhst/intro_to_nhst.md

```diff
@@ -0,0 +1,488 @@
+
+What counts as your population of interest depends on how you want to generalize your results: 
+
+- If you want to learn something about how pediatric patients in general (not only in your hospital) fared, then the data you have from your hospital is a (non-random) sample from a larger population (all kids who got Covid-19 in the first year of the pandemic). 
+- Alternatively, maybe you're only interested in patients at your hospital specifically, but your questions are about how patients at your hospital might fare in pandemic events in general, not just specifically the Covid-19 pandemic --- in that case, your data are again a (non-random) sample. 
+- It's possible, though, that your question is truly just about how patients at your hospital fared during this specific event, and you're not trying to extrapolate. In that case, your data represent the whole population of interest, not a sample. 
+
+<div class = "important">
+<b style="color: rgb(var(--color-highlight));">Important note</b><br>
+
+For most research questions, it's impossible to test the full population, and it's also impossible to get a true random sample from it. 
+It's still an important intellectual exercise to identify what your theoretical population of interest is, so that you can think clearly about the limitations of your sample to inform you about that population (and maybe even identify ways to collect a more useful sample!). 
+
+</div>
+
+## The null hypothesis
+
+Often, researchers are curious about an effect, but they usually have only a general scientific hypothesis rather than a precise statistical hypothesis. 
+
+For example, let's say a reseacher is interested in predictors of asthma attacks and has identified an environmental toxin that may be associated with more attacks. 
+The researcher's scientific hypothesis would be something like "patients with exposure to X will have more asthma attacks than patients without exposure to X." 
+Although that's a perfectly good scientific hypothesis, it's not precise enough to test statistically because "more" could mean any of a range of effects (a slight increase, a huge increase, or anything in between).
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For a great discussion of the difference between scientific and statistical hypotheses, see this [post on null hypothesis significance testing](https://education.arcus.chop.edu/null-hypothesis-testing/). 
+
+</div>
+
+A statistically precise hypothesis would be something like "patients with exposure to X will have an average of 2.6 more asthma attacks per year than patients without exposure to X," but that's not really useful scientifically --- if you end up finding that they have an average of 3.5 more attacks per year instead of 2.6, does that mean your hypothesis was wrong?
```

Comment:
```suggestion
A statistically precise hypothesis would be something like "patients with exposure to X will have an average of 2.6 more asthma attacks per year than patients without exposure to X," but that's not really useful scientifically-- if you end up finding that they have an average of 3.5 more attacks per year instead of 2.6, does that mean your hypothesis was wrong?
```